source/Octopus.Client/IOctopusCommonAsyncRepository.cs

```diff
@@ -9,5 +9,6 @@ public interface IOctopusCommonAsyncRepository
         ITeamsRepository Teams { get; }
         IScopedUserRoleRepository ScopedUserRoles { get; }
         IUserPermissionsRepository UserPermissions { get; }
+        ICommunityActionTemplateRepository CommunityActionTemplates { get; }
```

Comment:
This means you can go `client.ForSystem().Install(...)` and it will always fail at runtime.

To make this fail at compile time involves a breaking change, so it needs further thought. It might still be worth doing, in which case we can do this as a separate PR.

Reply:
To be clear, the breaking change would involve splitting up the community action template repository into two repositories. See https://github.com/OctopusDeploy/OctopusClients/commit/225b8e302069d67f5ecc320e19e05403a407f4dd

Reply:
Good catch